### PR TITLE
Adding C# Switch concept

### PR DESCRIPTION
**Pull request by kylestarr**
### Description

I'm adding the topic of C# Switch Statements, file `/content/c-sharp/concepts/switch/switch.md`. I am currently enrolled in the iOS Developer Career Path. Contributing to the Codecademy Docs Repository is part of the curriculum.

Closes: #1733 

Revised PR of https://github.com/Codecademy/docs/pull/1788

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by kylestarr** [comment]
@caupolicandiaz @SSwiniarski Here's a revised PR from a valid GitHub account / email address. Refer to https://github.com/Codecademy/docs/pull/1788

**Review by caupolicandiaz — APPROVED** [review]
@kylestarr LGTM.

File: content/c-sharp/concepts/switch/switch.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    break;
+  default:
+    // Code block
+    break;
+}
+```
+
+A `switch` statement includes one or more `case` conditions. The `case` conditions are evaluated in top-to-bottom order. If none of the `case` conditions evaluate to `true`, the `default` condition is executed. If no `default` is defined, the switch statement is bypassed.
+
+## Properties
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/switch/switch.md:38]: Remove this header, these aren't actually "properties" and they're elaborating on the syntax. I'd replace this with a sentence introducing the bulleted list.

File: content/c-sharp/concepts/switch/switch.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+- `switch`: Definition of the `switch` statement.
+- `expression`: The logic each `case` will be evaluated against.
+- `case`: A value the `expression` is matched against. If the `case` matches the result of the `expression`, the code block within the `case` will be executed.
+- `break` (recommended): Terminates the switch statement. Recommended at the end of each `case` and `default` to terminate the switch once the case has been fulfilled. A switch statement tests all cases unless a `break`, `throw`, or `return` statement is used to direct control out of the switch. If no `break` statement exists in a case, C# will throw a compile time error.
+- `default` (optional): The behavior executed if no `case` condition matches the `expression`.
+
+## Codebyte Example
+
+```codebyte/csharp
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/switch/switch.md:48]: Have an introductory sentence before the codebyte block explaining what it's doing.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@kylestarr, I did a first review and left a couple of comments.

File: content/c-sharp/concepts/switch/switch.md (hunk 1 above)
Comment (kylestarr) [reply content/c-sharp/concepts/switch/switch.md]: Updated

File: content/c-sharp/concepts/switch/switch.md (hunk 2 above)
Comment (kylestarr) [reply content/c-sharp/concepts/switch/switch.md:52]: Updated

**Comment by kylestarr** [comment]
@SSwiniarski I've incorporated your recommended edits. Thanks!

File: content/c-sharp/concepts/switch/switch.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+- `expression`: The logic each `case` will be evaluated against.
+- `case`: A value the `expression` is matched against. If the `case` matches the result of the `expression`, the code block within the `case` will be executed.
+- `break` (recommended): Terminates the switch statement. Recommended at the end of each `case` and `default` to terminate the switch once the case has been fulfilled. A switch statement tests all cases unless a `break`, `throw`, or `return` statement is used to direct control out of the switch. If no `break` statement exists in a case, C# will throw a compile time error.
+- `default` (optional): The behavior executed if no `case` condition matches the `expression`.
+
+## Codebyte Example
+
+The following example creates a string variable named `favoriteTurtle` with the value of `"Donatello"`. A switch statement is defined that evalutes the value of `favoriteTurtle`. If a case matches the value of `favoriteTurtle`, a corresponding message is printed to the console. If none of the cases in the switch statement match the value of `favoriteTurtle`, a default message is printed to the console.
+
+Let's see how well you know '80s cartoon theme songs!
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/switch/switch.md:50]: Let's remove this line.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@kylestarr One more thing.

File: content/c-sharp/concepts/switch/switch.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+Title: 'Switch'
+Description: 'Switch statements execute a code block from a list of conditions if a condition is true.'
+Subjects:
+  - 'Code Foundations'
+  - 'Web Development'
+  - 'Mobile Development'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Control Flow'
+  - 'Switch'
+  - 'C#'
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/switch/switch.md:11]: Can you remove the C# tag. We generally don't have language topics as Tags as they're redundant.

File: documentation/tags.md [hunk 5]
```diff
@@ -48,6 +48,7 @@ Catch
 Characters
 Chatbots
 Cryptocurrency
+C#
```
Comment (SSwiniarski) [inline documentation/tags.md:51]: I'd remove this as we generally don't have language topics as tags as they're redundant.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@kylestarr Just two more.

File: content/c-sharp/concepts/switch/switch.md (hunk 4 above)
Comment (kylestarr) [reply content/c-sharp/concepts/switch/switch.md]: Fixed

File: documentation/tags.md (hunk 5 above)
Comment (kylestarr) [reply documentation/tags.md]: Fixed

**Review by SSwiniarski — APPROVED** [review]
@kylestarr, Looks good to merge

**Comment by SSwiniarski** [comment]
@kylestarr Congrats on your new entry. The live version is here:https://www.codecademy.com/resources/docs/c-sharp/switch
You're currently listed as an anonymous contributor. If you want to be credited follow the instructions below to add your GitHub username to your Codecademy profile. If not, there is nothing more you need to do.

image

To add your GitHub username to your Codecademy profile:

1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:

image

2.) Then log out and log back into Codecademy using your GitHub login:

image

**Comment by kylestarr** [comment]
Thanks for your patience and guidance, @SSwiniarski and @caupolicandiaz!

**Pull request merged**